src/worker/db/getCopies/playersPlus.test.ts

```diff
@@ -299,10 +299,11 @@ describe("worker/db/getCopies/playersPlus", () => {
 		}
 
 		// This will break if ovr + fuzz is over 100 (should check bounds), but that never happens in practice
-		assert.strictEqual(
-			pf.ratings.ovr,
-			Math.round(p.ratings[1].ovr + p.ratings[1].fuzz),
-		);
+		// fuzz is not this simple anymore
```

Comment:
Should just be deleted rather than commented out

Reply:
My thinking was I should add tests. Check for decent bounds. Check that the +/- functions behave correctly. I don't think losing tests make sense, especially when the code just got more complicated. I should just write some additional replacement test.